src/pages/nft/gift/index.vue

```diff
@@ -4,7 +4,15 @@
       <NFTWidgetBaseCard>
         <NuxtLink
           :to="
-            localeLocation({ name: 'nft-class-classId', params: { classId } })
+            collectionId
+              ? localeLocation({
+                  name: 'nft-collection-collectionId',
+                  params: { collectionId },
+                })
+              : localeLocation({
+                  name: 'nft-class-classId',
+                  params: { classId },
+                })
```

Comment:
Can be put this in `nftOrCollectionMixin` and reuse it in `handleClickViewClass`?

Reply:
added but not sure how to name this

Reply:
I think is fine for now